cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -199,14 +200,14 @@ public KafkaAssemblyOperator(Vertx vertx, boolean isOpenShift,
         private final Kafka kafkaAssembly;
         private final Reconciliation reconciliation;
 
-        private ClusterCa clusterCa;
+        ClusterCa clusterCa; /* test */
```

Comment:
Since you're commenting on the (absence) of visibility modifier I think it's better to emphasize that by putting the comment in the same position as the (missing) visibility modifier:

```java
/*test*/ ClusterCa clusterCa;
```